packages/trpc/server/routers/_app.ts

```diff
@@ -2,16 +2,19 @@
  * This file contains the root router of your tRPC-backend
  */
 import { router } from "../trpc";
-import { viewerRouter } from "./viewer/_router";
+import { getViewerRouter } from "./viewer/_router";
+
+export { AppRouter } from "./viewer/_router.types";
 
 /**
  * Create your application's root router
  * If you want to use SSG, you need export this
  * @link https://trpc.io/docs/ssg
  * @link https://trpc.io/docs/router
  */
-export const appRouter = router({
-  viewer: viewerRouter,
-});
-
-export type AppRouter = typeof appRouter;
+export async function createAppRouter() {
```

Comment:
- Only used in `ssgInit()` in `ssg.ts` at the moment (but the catch is that `ssgInit()` isn't used anywhere atm)
- Hence, technically, we can remove both `ssgInit()` and `createAppRouter()`